NEWS.rst

```diff
@@ -2,6 +2,26 @@
 Release Notes
 =============
 
+pkgdev 0.2.1 (2022-05-21)
+-------------------------
+
+- pkgdev: Add configuration support. For more info look at [#]_.  (Arthur
+  Zamarin, #48, #62)
+
+- pkgdev commit: new summary for stabilizing ALLARCHES (Arthur Zamarin, #61)
+
+- pkgdev mask: offer to send last-rite message email to gentoo-dev ML when
+  last-riting a package (Arthur Zamarin, #63)
+
+- pkgdev manifest: add --if-modified (Arthur Zamarin, #66)
+
+- pkgdev manifest: add --ignore-fetch-restricted (Arthur Zamarin, #67)
+
+- pkgdev commit: **BREAKING-CHANGE** make sign-off off by default
```

Comment:
1. Let's put this first so it's harder to miss?
2. change phrasing to "disable sign-off by default" (avoid double "off", and ambiguous "make" (no implied "direction"))

Reply:
Thanks.